### DISCUSSION TITLE
Is there a write-up of the community call?

### DISCUSSION THREAD

**Discussion by rik** [mdn/mdn-community#424] [Announcements]
I could not attend [the community call](https://github.com/mdn/yari/issues/9208#issuecomment-1625683795) but I'd love to follow the conversations.

**Comment by caugner** [comment]
The answers are being added to the discussions.

**Reply by DavidJCobb** [reply]
Has the situation changed? [At least one discussion](https://github.com/orgs/mdn/discussions/414#discussioncomment-6401616) has gone unanswered for about two weeks now, and you've hidden your own comment stating that it would get an answer.

**Comment by rik** [comment]
I may have misphrased my ask: are there any notes from the call ?

**Reply by caugner** [reply]
As announced in the invitation, we're sharing the answers to the questions asked. We don't have any additional notes to share.
